### PR TITLE
fix: Indicate updatable status of deployment

### LINKED ISSUE #90: Indicate if a deployment was done with a different (non-console) client
**Is your feature request related to a problem? Please describe.**
Users sometimes attempt to modify deployments not done using console. Example https://discord.com/channels/747885925232672829/1053359341144653957/1089949113916850227
Users need to be able to quickly know (and ideally filter) which deployments were done using console vs. other clients

**Describe the solution you'd like**
I can think or two ways to handle this:
1. Add some messaging (could be an icon or label next to the deployment name) -- this is the easiest but doesn't allow sorting or filtering)
2. Column to indicate if a deployment was not done using console - a "check" or something indicates that a different client was used - probably a little more work but will let the user sort and quickly see all deployments done with and without console
3. Filtering option (when we do support filters on this page).

FINAL DECISION/ DESIGN (ignore the "filter" and "view" UI):

image

** Implementation hint **
Check to see if we have the SDL. If we do then Console did the deployment. If not, some other client did.

**Describe alternatives you've considered**
Do nothing.

**Additional context**
There are multiple (vertically focused) clients for deploying on Akash: https://github.com/akash-network/community/tree/main/sig-clients

### PR DESCRIPTION
**Pull request by Amit0617**
fixes #90

### CODE REVIEW

**Comment by anilmurty** [comment]
Thanks for this @Amit0617 - have you tested a deployment after this change? email me if you need $AKT to test and post a recording of the deployment with this change please.

**Comment by Amit0617** [comment]
Sent email

**Comment by anilmurty** [comment]
10 $AKT sent

**Comment by Amit0617** [comment]
Screenshot_20230422_215046
Screenshot_20230422_215110

I didn't found any component for tooltip having white background. So it is in it's default state.

**Comment by anilmurty** [comment]
Thanks @Amit0617 - have you verified deploying a workload (tetris is fine) and closing out?

**Comment by Amit0617** [comment]
I didn't got it clearly, do you want me to delete the deployment?

**Comment by anilmurty** [comment]
Create a new Tetris deployment (Fun & Games > Tetris > Deploy), confirm successful deployment (should be able to click URL at the end and see app) and then delete deployment.

**Comment by anilmurty** [comment]
And post screen recording if possible

**Comment by Amit0617** [comment]
https://user-images.githubusercontent.com/71893015/233838317-0ab192e5-9e31-4883-93db-864cf6ba4383.mp4

**Comment by Amit0617** [comment]
In between transaction fails when I was trying to save some $AKT as I had already got to less than 5 $AKT after first deployment, so I was concerned if in between, funds get shorted but I realized you get a big portion back if you delete the deployment.

**Comment by anilmurty** [comment]
@Amit0617 - thanks for the screen recording. Yes, an escrow (with default 5 $AKT) is created when you create a lease with a provider. The left over funds (when you close the lease) are returned back to your wallet when the lease is closed.

This looks good to me to review and merge if OK @jtary - please review

File: web/src/components/DeploymentTable/index.tsx [hunk 1]
```diff
@@ -124,7 +135,13 @@ function DeploymentTableHead(props: DeploymentTableHeadProps) {
               direction={order}
               onClick={createSortHandler(headCell.id)}
             >
-              {headCell.label}
+              {headCell.label === 'UPDATABLE'
+                ? (<>
+                  {headCell.label}
+                  <Tooltip title="If the initial deployment was done using a different client(not console) it cannot be updated by Console." placement="bottom">
```
Comment (anilmurty) [inline web/src/components/DeploymentTable/index.tsx]: add space after client:
"using a different client (not console)"

**Review by jtary — APPROVED** [review]

**Pull request merged**